vmdb/tools/fix_auth/models.rb

```diff
@@ -17,8 +17,18 @@ class FixMiqDatabase < ActiveRecord::Base
     self.table_name = "miq_databases"
     self.password_columns = %w(registration_http_proxy_server session_secret_token csrf_secret_token)
 
-    def self.hardcode(old_value, _new_value)
-      super(old_value, SecureRandom.hex(64))
```

Comment:
Hmm. Think re-introducing this will fix the bug as well. Think this will read better

since it was so simple, it was merged into `recrypt:
```ruby
def hardcode(_old_value, new_value)
  MiqPassword.encrypt(new_value)
end
```